wgsl/index.bs

```diff
@@ -1072,6 +1072,9 @@ An <dfn>integer literal</dfn> is:
     * `0x` or `0X` followed by a sequence of hexadecimal digits.
 * Then an optional `i` or `u` suffix.
 
+Note: A leading zero on a non-zero number literal (e.g. 012) is forbidden,
```

Comment:
Small feedback: it might be good to note as integer literal than number because floats allow it when there is dot. Thank you!

Reply:
Done!